### ISSUE TITLE
Create a license matrix for ArcMap and ArcGIS Pro

### ISSUE THREAD

**Issue by topowright** [Esri/military-tools-desktop-addins#12]
### Tasks to be done June 13
- [ ] Add statement for licensing for each component 
- [ ] Add a matrix table to Military Tools @topowright

We currently need to understand what licenses a user needs to run each part and the entire product. This needs to be done for ArcMap and ArcGIS Pro. 

### Expected Behavior
A user should understand by looking at one chart what they will need in order to use the product. 

### Current Behavior
No Chart

### Your Environment

* Addin Version used: 
* Environment name and version (e.g. ArcMap 10.3, Pro 1.4):
* Operating System and version (desktop or mobile):
* Link to your project/test data (if applicable):

**Comment by ACueva** [comment]
@topowright @dfoll Any suggestions where the final matrix created should live?  Perhaps expand the requirements section? http://solutions.arcgis.com/defense/help/military-tools/

**Comment by ACueva** [comment]
I've tentatively created a wiki page with the extension matrix information: https://github.com/Esri/military-tools-desktop-addins/wiki/Extensions-Matrix
The steps I'm using to verify the correct extension requirements is to load ArcGIS Pro/ArcMap with the lowest license level and no extensions.  I enable additional license levels and extensions only when the particular component requests it or is not functioning.

**Comment by ACueva** [comment]
I've identified a crash in Distance and Direction.  Working on reproducing the issue.  Unknown at this time if it related to the license level.

**Comment by topowright** [comment]
@ACueva do we understand the crash issue for this item?

**Comment by ACueva** [comment]
@topowright I am unable to reproduce this issue on 2.1.  Working with @csmoore we found that the bglcache.dll was the cause of the crash on the ArcGIS Pro 2.0 machine I tested against. (This appears to be a ArcGIS Pro core dll).  I tried to reproduce this issue on a different ArcGIS Pro 2.0 machine and was unable to reproduce.  I'm thinking this is a machine specific issue.

**Comment by lfunkhouser** [comment]
Moving this to the Product Backlog. This is not part of the scope for the current Release for January 2018.

**Comment by topowright** [comment]
I am working on the matrix for all MT Tools now. I am about half way thru. Seeing some issues with the way tools are designed that can be revisited later on. Currently GRG from Point does not work on Basic.

**Comment by BobBooth** [comment]
Fleshed out license matrix table, fixed typos, tool names, tested and added info for rows with no X, split table into sections for add-ins/toolsets.
I cannot remove tags in this repo.
@topowright @lfunkhouser  @dfoll

**Comment by BobBooth** [comment]
@dfoll - ready to add to document map(s)

**Comment by dfoll** [comment]
added to document map, republished. looks good on DEV waiting on a push to STG. new build is building now, closing after we see the proper version is in the .pdf
